src/script/tracking/UserData.ts

```diff
@@ -18,8 +18,8 @@
  */
 
 export const UserData = {
-  CONTACTS: 'user.contacts',
-  IS_TEAM: 'team.is_team',
-  TEAM_SIZE: 'team.team_size',
-  USER_TYPE: 'team.user_type',
+  CONTACTS: 'user_contacts',
+  IS_TEAM: 'team_is_team',
+  TEAM_SIZE: 'team_team_size',
+  USER_TYPE: 'team_user_type',
```

Comment:
Are these changes part of logarithmic rounding?

Reply:
nope, these are updates by Eva

Reply:
Shouldn't we then apply them in a different PR?

Reply:
done in https://github.com/wireapp/wire-webapp/pull/9581/commits/bec95acaa144272580fa0d1841b782a5e477f9e1